### ISSUE TITLE
Underflow handling

### ISSUE THREAD

**Issue by lairez** [JuliaIntervals/IntervalArithmetic.jl#302]
I am no expert in interval arithmetic, but the following results seems like mishandling of underflows:

```julia
i = interval(1e-200)
i == 0      # false
i*i == 0    # true
```
The fundamental inclusion property of interval arithmetic is broken here... What does the standard says about this?

**Comment by dpsanders** [comment]
This is a bug. It's related to the current default implementation using error-free arithmetic, which needs to be fixed for these underflow cases. Thanks for reporting.

**Comment by dpsanders** [comment]
The correct solution should be
```
Interval(0.0, nextfloat(0.0))
```

**Comment by dpsanders** [comment]
This has since been fixed in version 0.17.
